_posts/2019-08-26-how-to-use-gruntwork-service-catalog.adoc

```diff
@@ -0,0 +1,1934 @@
+  <<automated_tests_terraform>>.
+. Deploy your changes to each environment. You do this using the same process outlined in <<deploy_terraform>>.
+
+[[using_scripts_binaries]]
+=== Using scripts and binaries
+
+This section will show you how to use scripts and binaries from the Gruntwork Service Catalog. As illustrative examples,
+we'll deploy the `ip-lockdown` script and `ssh-grunt` binary from
+https://github.com/gruntwork-io/module-security[module-security].
+
+IMPORTANT: You must be a https://gruntwork.io/[Gruntwork subscriber] to access `module-security`.
+
+ip-lockdown::
+  `ip-lockdown` is a Bash script you can use to lock down specific outgoing IP addresses on a Linux server so only
+  specific users can access them. The main motivation is to lock down the
+  https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html[metadata endpoint] on EC2 instances so
+  only specific users (e.g., root) can access it—and whatever IAM permissions and other secrets it confers—rather than
+  just any user who happens to get access to EC2 instance.
+
+ssh-grunt::
+  `ssh-grunt` is a binary (compiled from Go code) that allows you to manage SSH access to your EC2 instances using an
+  identity provider such as AWS IAM or any SAML-based identity provider (e.g., Active Directory, Google, Okta, etc.).
+  This allows you to grant access to specific EC2 instances by putting developers into certain groups (e.g., into
+  specific IAM groups or AD groups) and for each developer to SSH to EC2 instances using their own username and SSH
+  key.
+
+Let's walk through how to use these two tools.
+
+==== Gruntwork Installer
+
+The easiest way to install scripts and binaries from the Gruntwork Service Catalog is to use the
+https://github.com/gruntwork-io/gruntwork-installer[Gruntwork Installer]. This is an open source Bash script that
+allows you to install scripts and binaries via a one-liner, similar to `apt-get install` or `yum install` (if you're
+curious why we don't use `apt` or `yum` directly, see
+https://github.com/gruntwork-io/gruntwork-installer#motivation[Gruntwork Installer Motivation]).
+
+The first step is to install the Gruntwork Installer itself, which you can do as follows (make sure to replace
+`<VERSION>` below with the latest version from the
+https://github.com/gruntwork-io/gruntwork-installer/releases[Gruntwork Installer releases page]):
+
+----
+curl -LsS https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash /dev/stdin --version <VERSION>
+----
+
+Now you can install any script `XXX` at version `YYY` from repo `ZZZ` of the Gruntwork Service Catalog as follows:
+
+----
+gruntwork install --module-name XXX --tag YYY --repo ZZZ
+----
+
+And you can install any binary `XXX` at version `YYY` from repo `ZZZ` of the Gruntwork Service Catalog as follows:
+
+----
+gruntwork install --binary-name XXX --tag YYY --repo ZZZ
+----
+
+Note that if repo `ZZZ` is a private GitHub re
```

Comment:
I have a feeling this section could go in a contributing.md somewhere and just be linked here instead of being in the main body of this article

Reply:
Same thing here... Which section?

Reply:
Ah weird! For some reason looks like the comment got attached to the wrong bit. This comment refers to the section on contributing to the service catalog